### PR TITLE
Sg 17/ionicons

### PR DESCRIPTION
**Pull request by adarsh-murthy**
Adds all the ion icons as per figma. 
Please load up the example app to make sure all icons look good.

# To Test

```
cd example
yarn
yarn start
```
go to `localhost:8080` to access the app.

# Screenshot of icons
Screen Shot 2020-08-03 at 12 16 38 PM

Screen Shot 2020-08-06 at 1 29 16 PM

### CODE REVIEW

**Comment by herrethan** [comment]
can you include a screenshot of how these are rendered in the example? also would be useful to have some example tests for sizing the icons.

- convention wise, i think it would be wise to prefix all these with `Icon`, as in `IconLightBulb` etc. 
- maybe it makes more sense to create our own standard wrapper for SvgIcon, so we don't need to redefine viewBox everytime.
- what's up with viewBox always being `0 0 24 25`?

**Comment by adarsh-murthy** [comment]
@herrethan I gotchu! Quick follow up:
- What do you mean create our own standard wrapper? Can you please elaborate a little?
I don't know why the view box is `0 0 24 25` for most of them.

**Comment by herrethan** [comment]
@adarsh-murthy ok let's check with joel. 24/25 sounds wrong.

File: src/ionicons/solidIcons/IconCodeDownload.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export default function IconCodeDownload(props: SvgIconProps) {
+  return (
+    <SvgIcon viewBox="0 0 24 25" fill="none" {...props}>
```
Comment (herrethan) [inline src/ionicons/solidIcons/IconCodeDownload.tsx:6]: 👀

File: src/ionicons/solidIcons/IconCodeSlashSolid.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export default function IconCodeSlashSolid(props: SvgIconProps) {
+  return (
+    <SvgIcon viewBox="0 0 24 25" fill="none" {...props}>
```
Comment (herrethan) [inline src/ionicons/solidIcons/IconCodeSlashSolid.tsx:6]: 👀

File: src/ionicons/solidIcons/IconCheckmarkCircleSolid.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export default function IconCheckmarkCircleSolid(props: SvgIconProps) {
+  return (
+    <SvgIcon viewBox="0 0 20 20" fill="none" {...props}>
```
Comment (herrethan) [inline src/ionicons/solidIcons/IconCheckmarkCircleSolid.tsx:6]: is this one unfixable?

**Comment by herrethan** [comment]
this is looking great @adarsh-murthy. on the font size inheritance issue, can you try adding `fontSize="inherit"`and see if this works? i think we may have full support for what we need already.

File: example/app.js [hunk 4]
```diff
@@ -98,6 +174,94 @@ const HelloWorld = () => {
...
+          <IconShareSocialSolid />
+          <IconSpreadsheetSolid />
+          <IconSquareSolid />
+          <IconTodaySolid />
+          <IconTrashSolid />
+          <IconWarningSolid />
+        </div>
+        <h1>I am a heading 1 with icon <IconTodaySolid /></h1>
+        <h2>I am a heading 2 with icon <IconTodaySolid /></h2>
+        <h3>I am a heading 3 with icon <IconTodaySolid /></h3>
```
Comment (herrethan) [inline example/app.js:258]: try
```js
<h3>I am a heading 3 with icon <IconTodaySolid fontSize="inherit" /></h3>
```

**Pull request merged**